JS-Basic/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "JS-Basic",
+  "version": "1.0.0",
+  "description": "",
+  "main": "apt.js",
+  "scripts": {
+    "test": "jest"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "moment": "^2.29.1"
```

Comment:
better to freeze all libs versions due to security reasons. Somebody can hack npm package and upload patch version with virus and your npm install latest patch version if you use '^'